client/me/help/help-unverified-warning/index.jsx

```diff
@@ -28,57 +29,52 @@ class HelpUnverifiedWarning extends Component {
 	render() {
 		const { resendState } = this.state;
 
-		const resendEmail = () => {
-			this.setState( {
-				resendState: RESEND_IN_PROGRESS,
-			} );
-
-			userFactory().sendVerificationEmail()
-				.then( () => this.setState( { resendState: RESEND_SUCCESS } ) )
-				.catch( () => this.setState( { resendState: RESEND_ERROR } ) );
-		};
-
 		const resendStateToMessage = ( resendState ) => {
 			switch ( resendState ) {
 				case RESEND_IDLE:
 					return this.props.translate( "Trouble activating your account? Just click this button and we'll resend the activation for you." );
 				case RESEND_IN_PROGRESS:
-					return this.props.translate( 'Sending…' );
+					return '';
 				case RESEND_SUCCESS:
-					return this.props.translate( "Please check your email for an activation email and click the link to finish your signup. If you don't receive the email within a few minutes, please be sure to check your spam folders." );
+					return this.props.translate( "Please check your email for an activation email and click the link to finish your signup." );
 				case RESEND_ERROR:
 					return this.props.translate( "Sorry that we've encountered an error on sending you an activation email. Please try again later." );
 				default:
 					return 'Unknown activation email resending state.';
 			}
 		};
 
-		const resendStateToNoticeType = ( resendState ) => {
-			switch ( resendState ) {
-				case RESEND_IDLE:
-					return 'is-warning';
-				case RESEND_IN_PROGRESS:
-					return 'is-info';
-				case RESEND_SUCCESS:
-					return 'is-success';
-				case RESEND_ERROR:
-					return 'is-error';
-				default:
-					return 'is-error';
-			}
+		const resendEmail = () => {
+			this.setState( {
+				resendState: RESEND_IN_PROGRESS,
+			} );
+
+			userFactory().sendVerificationEmail()
+				.then( () => {
+					const nextResendState = RESEND_SUCCESS;
+
+					this.setState( { resendState: nextResendState } )
+					notices[ 'success' ]( resendStateToMessage( nextResendState ) );
```

Comment:
Alternatively, the notices [can be triggered using Redux actions](https://github.com/Automattic/wp-calypso/tree/master/client/notices)